finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/AccessLogger.scala

```diff
@@ -0,0 +1,75 @@
+package com.twitter.finagle.buoyant.h2
+
+import com.twitter.finagle.{Service, SimpleFilter}
+import com.twitter.finagle.context.RemoteInfo
+import com.twitter.logging._
+import com.twitter.util._
+import java.net.InetSocketAddress
+
+object AccessLogger {
+  def mk(
```

Comment:
This method which processes option properties and applies defaults feels out of place in a pure library like this.  I think this logic would fit better directly in MeshIfaceInitializer.